department-of-veterans-affairs/caseflow
# Updated the format of errors in the BuildScheduleUploadContainer

Resolves #15660 

### Description

Refactors the way errors are formatted in the hearings build schedule workflow. Previously there were multiple different formats that the schedule periods controller was responding with ([see this documentation](https://github.com/department-of-veterans-affairs/caseflow/wiki/Error-Handling)).

This PR changes the format to be what is described in the ticket above.

### Acceptance Criteria
- [ ] Errors display correctly inline with form inputs

### Testing Plan

1. Impersonate user `BVASYELLOW`
1. The Schedule Periods controller: `app/controllers/hearings/schedule_periods_controller.rb`
1. In the `create` function (line 33) add the line `fail ActiveRecord::RecordInvalid`
1. Navigate to the build upload page: http://localhost:3000/hearings/schedule/build/upload
1. Select the `RO/CO` option and provide start and end dates as well as a valid excel document
1. Click the `continue` button and wait for the form to process
1. Ensure that you receive a `Record Invalid` error in the form
1. Repeat the steps above for the `Judge` option
